mailbox/zoo-seq-provider/src/test/java/org/apache/james/mailbox/store/mail/ZooUidProviderTest.java

```diff
@@ -137,73 +137,72 @@ public boolean equals(Object obj) {
 
     }
 
+    private static TestingServer testServer;
```

Comment:
indentation fix, no code change.

Reply:
next time, please try to do a separate commit for this